workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx

```diff
@@ -239,25 +236,34 @@ const TeamMembersTable = ({
 	);
 
 	const checkIsValidRole = (userAccount) => {
-		const isInvalidRole = (role) => {
-			const invalidRoles = ['Security', 'Data', 'Critical'];
+		const isIncidentContactRole = (role) => {
+			const incidentRoles = ['Security', 'Data', 'Critical'];
 
-			return invalidRoles.some((keyword) => role.name.includes(keyword));
+			return incidentRoles.some((keyword) => role.name.includes(keyword));
 		};
 
 		const roles = getCurrentRoleBriefs(userAccount.selectedAccountSummary);
 
 		if (!roles.length) {
 			return ['User'];
-		} else {
-			return roles.map((role) => {
-				if (!isInvalidRole(role)) {
-					return role?.name;
+		}
+
+		const memberRoles = [];
+		let hasIncidentContact = false;
+
+		roles.forEach((role) => {
+			if (isIncidentContactRole(role)) {
+				if (!hasIncidentContact) {
+					memberRoles.push('Incident Contact');
```

Comment:
maybe we can just do a generic check if the array already has the value so there's never any duplicates?

Reply:
User may show twice again later if we add more roles but forget to change this code

Reply:
The way I've done it is already preventing duplication from happening. If he has more than one CIC role, “Incident Contact” will only appear once, in addition to the other non-CIC roles.

Reply:
I was thinking like this, what do you think?
```
		roles.forEach((role) => {
			let roleName = role?.name;

			if (isIncidentContactRole(role)) {
				roleName = 'Incident Contact';
			}
	
			if (!memberRoles.includes(roleName)) {
				memberRoles.push(roleName);
			}
		});

```

Reply:
Oh, the result is the same, but I think it is better this way, I'll send a correction.

Reply:
Done.